web/src/components/pages/Login/component.tsx

```diff
@@ -42,17 +46,20 @@ export const LoginPage = (): JSX.Element => {
     let navigate = useNavigate();
 
     const api = new Api();
+    const dispatch = useDispatch();
 
     const login = async () => {
-        setIsLoading(true);
         try {
-            // TODO save user data to reducer
-            const userData = (await api.post('url..', form)) as User;
+            const userData = (await api.post('/login', form)) as User;
```

Comment:
use generic types for POST
we want to declare with post requset will return

//
post method in api class should return Promise<T>